guide.js

```diff
@@ -309,7 +309,7 @@ function updateIframe(visible){
 			myframe.name="myframe";
 			myframe.id="myframe";
 			myframe.classList.add("iframe");
-			myframe.src="toc-3.html";
+			myframe.src="help.html";
```

Comment:
I like this as it now works for v3 and v4 guide

Reply:
Yeah I noticed the toc had to be set for each revision and I already had the idea for the help to be it's own tab so this sort of inadvertently killed two birds with one stone